### DISCUSSION TITLE
[@types/asana] Types for @node-asana:v2

### DISCUSSION THREAD

**Discussion by zgurea1** [DefinitelyTyped/DefinitelyTyped#66283] [Issues with a @types package]
Recently dev team from asana updated their js sdk to version 2, and change how the request are made. 
The @types/asana are not compatible with current version.

[a comment by an automated account is not shown]

**Comment by apiology** [comment]
Thanks @zgurea1 - good to know!  I don't personally have any immediate plans to write a new set of type definitions, but this would be a reasonable place to coordinate if anyone else does.
